.eslintrc

```diff
@@ -0,0 +1,98 @@
+{
+    "parser": "@typescript-eslint/parser",
+    "plugins": [
+      "react",
+      "@typescript-eslint"
+    ],
+    "extends": [
+      "eslint:recommended",
+      "plugin:react/recommended",
+      "airbnb",
+      "airbnb/hooks",
+      "airbnb-typescript"
+    ],
+    "rules": {
+      "comma-dangle": ["error", {
+        "objects": "always-multiline",
+        "arrays": "always-multiline",
+        "imports": "always-multiline",
+        "exports": "always-multiline",
+        "functions": "never"
+        }],
+      "max-len": ["warn", { "code": 120, "ignoreStrings": true }],
+      "no-use-before-define": "off",
+      "no-param-reassign": "off",
+      "no-redeclare": "off",
+      "no-undef": "off",
+      "no-underscore-dangle": "off", // We have _uid from Storyblok that can't be changed
+      "@typescript-eslint/no-redeclare": ["error"],
+      "object-curly-newline": ["error", { "consistent": true, "minProperties": 4 }],
+      "import/extensions": "off",
+      "no-shadow": "off",
+      "@typescript-eslint/no-shadow": ["error"],
+      "jsx-a11y/label-has-associated-control": [ 2, {
+        "components": [ "Label" ],
+        "required": {
+            "some": [ "nesting", "id" ]
+        }
+      }],
+      "space-before-function-paren": 0,
+      "react/jsx-props-no-spreading": 0,
+      "react/prop-types": 0,
+      "react/jsx-handler-names": 0,
+      "react/jsx-filename-extension": [
+        "error",
+        {
+          "extensions": [
+              ".js",
+              ".ts",
+              ".jsx",
+              ".tsx"
+          ]
+        }
+      ],
+      "react/jsx-fragments": 0,
+      "react/jsx-one-expression-per-line": 0,
+      "react/no-unused-prop-types": 0,
+      "react/require-default-props": 0,
+      "react/function-component-definition": [2, {
+        "namedComponents": ["function-declaration", "arrow-function"]
+      }],
+      "import/prefer-default-export": 0,
+      "import/no-extraneous-dependencies": 0,
+      "import/no-cycle": 0,
+      "import/no-unresolved": 0,
+      "no-unused-vars": 0,
+      "@typescript-eslint/no-unused-vars": [
+        "warn",
+        {
+          "vars": "local",
+          "args": "none"
+        }
+      ],
+      "import/export": 0,
+      "func-names": 0,
+      "semi": [1, "always"] // 1 is for warning
+    },
+    "env": {
+      "node": true,
+      "browser": true,
+      "jest": true
+    },
+    "globals": {
+      "JSX": true
+    },
+    "parserOptions": {
+      "ecmaVersion": 2020,
+      "ecmaFeatures": {
+        "legacyDecorators": true,
+        "jsx": true
+      },
+      "project": "./tsconfig.json"
```

Comment:
The `eslint-config-airbnb-typescript` plugin needs to know our TS config
https://github.com/iamturns/eslint-config-airbnb-typescript#4-configure-the-eslint-typescript-parser